src/components/RichTextEditor/Toolbar.tsx

```diff
@@ -58,13 +60,31 @@ export const Toolbar: FC = () => {
                 {...attributes.popper}
             >
                 <div className="tw-flex tw-items-center tw-px-3 tw-py-2 tw-bg-white tw-rounded tw-shadow-mid tw-gap-0.5">
-                    <BoldButton />
-                    <ItalicButton />
-                    <UnderlineButton />
-                    <StrikethroughButton />
-                    <CodeButton />
-                    <OrderedListButton />
-                    <UnorderedListButton />
+                    <InlineStyleButton style={InlineStyles.Bold}>
+                        <IconBold size={IconSize.Size16} />
+                    </InlineStyleButton>
+
+                    <InlineStyleButton style={InlineStyles.Italic}>
+                        <IconItalic size={IconSize.Size16} />
+                    </InlineStyleButton>
+
+                    <InlineStyleButton style={InlineStyles.Underline}>
+                        <IconUnderline size={IconSize.Size12} />
+                    </InlineStyleButton>
+
+                    <InlineStyleButton style={InlineStyles.Strikethrough}>s</InlineStyleButton>
+
+                    <BlockStyleButton blockType={BlockStyleTypes.Code}>
+                        <IconSnippet size={IconSize.Size16} />
+                    </BlockStyleButton>
```

Comment:
This is currently a inline style, ask Simone about adding a block level snippet (but then we would need language support)

Reply:
👍  nice catch, that's maybe something to have in the future. I'll bring it up in the next checkin. for now: d9e44836164baf2a12bc0b3cfac6e8b37231092f